### PR TITLE
Issue #1985: Migrate command `convert-to-db-storage`

### PR DESCRIPTION
**Pull request by zymap**
*Motivation*

- Use bkctl to run command `convert-to-db-storage`

*Modifications*

- #1985 
- Add command in `bookieGroup`

### CODE REVIEW

File: bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/Bookie.java [hunk 1]
```diff
@@ -627,6 +627,7 @@ private static LedgerStorage buildLedgerStorage(ServerConfiguration conf) throws
      */
     public static LedgerStorage mountLedgerStorageOffline(ServerConfiguration conf, LedgerStorage ledgerStorage)
             throws IOException {
+
```
Comment (jiazhai) [inline bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/Bookie.java]: nit: drop this change?

File: bookkeeper-server/src/main/java/org/apache/bookkeeper/tools/cli/commands/bookie/ConvertToDBStorageCommand.java [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+    /**
+     * Flags for this command.
+     */
+    @Accessors(fluent = true)
+    @Setter
+    public static class CTDBFlags extends CliFlags {
+
+        @Parameter(names = { "-l", "--ledgeridformatter" }, description = "Set ledger id formatter")
+        private String ledgerIdFormatter = "";
+
```
Comment (jiazhai) [inline bookkeeper-server/src/main/java/org/apache/bookkeeper/tools/cli/commands/bookie/ConvertToDBStorageCommand.java]: nit: dup empty lines

File: bookkeeper-server/src/main/java/org/apache/bookkeeper/tools/cli/commands/bookie/ConvertToDBStorageCommand.java [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+    /**
+     * Flags for this command.
+     */
+    @Accessors(fluent = true)
+    @Setter
+    public static class CTDBFlags extends CliFlags {
+
+        @Parameter(names = { "-l", "--ledgeridformatter" }, description = "Set ledger id formatter")
+        private String ledgerIdFormatter = "";
```
Comment (jiazhai) [inline bookkeeper-server/src/main/java/org/apache/bookkeeper/tools/cli/commands/bookie/ConvertToDBStorageCommand.java]: nit: how about put a more meaningful string in CliFlags? e.g. 
```
public static String NOT_INIT = "Flag Not Inited";
```
and then here could change into 
`private String ledgerIdFormatter = NOT_INIT;`

Comment (jiazhai) [reply bookkeeper-server/src/main/java/org/apache/bookkeeper/tools/cli/commands/bookie/ConvertToDBStorageCommand.java]: This is more readable for line 123 below.

**Comment by jiazhai** [comment]
Thanks @zymap for the help. left some nit comments

File: bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/Bookie.java (hunk 1 above)
Comment (zymap) [reply bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/Bookie.java]: fixed

File: bookkeeper-server/src/main/java/org/apache/bookkeeper/tools/cli/commands/bookie/ConvertToDBStorageCommand.java (hunk 2 above)
Comment (zymap) [reply bookkeeper-server/src/main/java/org/apache/bookkeeper/tools/cli/commands/bookie/ConvertToDBStorageCommand.java]: fixed

File: bookkeeper-server/src/main/java/org/apache/bookkeeper/tools/cli/commands/bookie/ConvertToDBStorageCommand.java (hunk 3 above)
Comment (zymap) [reply bookkeeper-server/src/main/java/org/apache/bookkeeper/tools/cli/commands/bookie/ConvertToDBStorageCommand.java]: Sure, have fixed

**Comment by sijie** [comment]
@zymap can you rebase your PR to latest master?

**Comment by zymap** [comment]
run bookkeeper-server bookie tests
run integration tests

**Pull request merged**